pallets/eth-bridge/src/lib.rs

```diff
@@ -223,8 +223,11 @@ pub mod pallet {
                 BoundedVec<(BoundedVec<u8, TypeLimit>, BoundedVec<u8, ValueLimit>), ParamsLimit>,
             caller_id: BoundedVec<u8, CallerIdLimit>,
         },
-        /// EventProcessed(bool, EthEventId)
-        EventProcessed {
+        EventProcessingAccepted {
```

Comment:
I would also re-use the same name as EthereumEvents: `EventAccepted` and `EventRejected`.